packages/app/cypress/e2e/sidebar_navigation.cy.ts

```diff
@@ -226,6 +226,53 @@ describe('Sidebar Navigation', () => {
       cy.get('[data-cy="app-header-bar"]').findByText('Settings').should('be.visible')
       cy.get('.router-link-active').findByText('Settings').should('be.visible')
     })
+
+    it('use saved state for nav size', () => {
+      cy.withCtx(async (ctx) => {
+        await ctx.actions.localSettings.setPreferences(JSON.stringify({ reporterWidth: 100 }))
+      })
+
+      cy.scaffoldProject('todos')
+      cy.openProject('todos')
+      cy.startAppServer()
+      cy.__incorrectlyVisitAppWithIntercept()
+
+      cy.contains('fixture.js').click()
+
+      cy.get('.toggle-specs-text').click()
+
+      cy.get('[data-cy="reporter-panel"]').invoke('outerWidth').then(($initialWidth) => {
+        expect($initialWidth).eq(100)
+      })
+    })
+
+    // NOTE: Remove skip when we fix cy.reload() in Cypress in Cypress
```

Comment:
We should create an issue for `cy.reload()` not working (if there isn't one already created) and link the issue here.

Reply:
https://cypress-io.atlassian.net/browse/UNIFY-1346

Reply:
Nice, could we add this ticket number and change note to TODO, to make it a little easier to put the pieces together later?